packages/backend-modules/migrations/script/db/create.js

```diff
@@ -1,80 +1,27 @@
 #!/usr/bin/env node
-const { paths } = require('@orbiting/backend-modules-env').config()
+require('@orbiting/backend-modules-env').config()
+const { createDB } = require('../../lib')
 
-const { PgDb } = require('pogi')
-const moment = require('moment')
-const Promise = require('bluebird')
-const fs = require('fs').promises
+async function main(DATABASE_URL) {
+  if (!DATABASE_URL) {
+    console.error('DATABASE_URL in .env is missing. You need to add it first.')
+    process.exit(1)
+  }
 
-const createUpdateEnv = (connectionString) => async (path) => {
   try {
-    const envData = await fs.readFile(path, 'utf8')
-    const updatedEnvData = envData.replace(
-      /^(DATABASE_URL=.+)$/gm,
-      `#$1\nDATABASE_URL=${connectionString}`,
+    console.error(`Creating database...`)
+    const dbName = await createDB(
+      DATABASE_URL,
+      process.env.NODE_ENV !== 'production',
     )
-
-    if (envData !== updatedEnvData) {
-      await fs.writeFile(path, updatedEnvData, 'utf8')
-      console.log(`DATABASE_URL in "${path}" updated.`)
-    }
+    console.error(`Database "${dbName}" created.`)
```

Comment:
`console.error` just causes the message to be logged to `stderr` instead of `stdout`. In other environments, it is common practice to print debug info to `stderr` instead of `stdout` so that `stdout` can be used for piping to different programs.

Because this message has no programmatic use and is only for human debug purposes, I decided printing to `stderr` is the right choice.